python/ray/__init__.py

```diff
@@ -46,6 +46,13 @@
             e.args += (helpful_message, )
     raise
 
+# Because Modin uses type hints, python2 would require stripping these hints.
```

Comment:
Is this still the case? We should strip the hints in the modin release .tar.gz tarball to make sure it works on python 2

Reply:
Yes, we could do that. I am working on our `strip_hints` script to be able to work here. Right now it is kind of involved.